content/08.transmission.md

```diff
@@ -1,24 +1,68 @@
+The extent to which aerosol transmission contributes to the spread of respiratory viruses is less clear.
+In influenza A, for example, viral particles can be detected in aerosols produced by infected individuals, but the extent to which these particles drive the spread of influenza A infection remains under debate [@doi:10.4415/ANN_13_02_03; @doi:10.1097/QCO.0000000000000563; @doi:10.3201/eid1301.061202; @doi:10.1038/s41598-019-38825-y; @doi:10.1093/femsre/fuv039].
+Regardless of its role in the spread of influenza A, however, aerosol transmission likely played a role in outbreaks such as the 1918 Spanish Influenza (H1N1) and 2009 "swine flu" (pH1N1) [@doi:10.1093/femsre/fuv039].
+Contact, droplet, and aerosol transmission are therefore all worth evaluating when considering possible modes of transmission for a respiratory virus like SARS-CoV-2.
+
+#### Transmission of HCoV
+
+All three of these mechanisms have been identified as playing a role in the transmission of HCoV [@doi:10.4415/ANN_13_02_03], including the highly pathogenic coronaviruses SARS-CoV and MERS-CoV [@doi:10.1016/B978-0-12-385885-6.00009-2; @doi:10.1038/nrmicro.2016.81].
+SARS transmission is thought to proceed primarily through droplet transmission, but aerosol transmission is also considered possible [@doi:10.4415/ANN_13_02_03], and fomite transmission may have also played an important role in some outbreaks [@doi:10.1371/journal.pone.0181558].
+Similarly, the primary mechanism of MERS transmission is thought to be droplets because inter-individual transmission appears to be associated with close interpersonal contact (e.g., household or healthcare settings), but aerosolized particles of the MERS virus have been reported to persist much more robustly than influenza A under a range of environmental conditions [@doi:10.2807/1560-7917.es2013.18.38.20590; @doi:10.1186/s12985-015-0439-5].
+While droplet-based and contact transmission were initially considered to be the primary modes by which SARS-CoV-2 spread [@doi:10.1038/s41368-020-0075-9], as additional information emerged, the possibility of aerosol transmission was also raised [@doi:10.1126/science.abc6197; @doi:10.1056/NEJMc2004973; @doi:10.1093/cid/ciaa939].
+For example, the detection of SARS-CoV-2 viral particles in air samples taken from hospitals treating COVID-19 patients led to the concern that the virus could also spread via aerosols [@doi:10.1038/s41586-020-2271-3].
+The stability of the virus both in aerosols and on a variety of surfaces appeared similar to that of SARS-CoV [@doi:10.1056/NEJMc2004973], and fomite transmission may play a role in transmission (e.g., [@doi:10.3201/eid2606.200412]).
+However, while the possibility of aerosol transmission seems plausible, the evidence suggests that droplet and contact transmission are the dominant mechanisms driving the spread of the virus [@doi:10.1001/jama.2020.12458].
```

Comment:
So the one thing I would say is that, while fomite transmission is possible, I am not aware of any specific incidences where it was proven. Hence why the emphasis of washing groceries and allowing take aways and food deliveries didn't become a major issue.

Reply:
I agree! Let me see if I can find some good sources on this and I'll tag you again. Thank you so much @RLordan!